app/experimenter/experiments/tests/factories.py

```diff
@@ -264,10 +265,11 @@ class Meta:
 
 
 class ExperimentChangeLogFactory(factory.django.DjangoModelFactory):
+
     experiment = factory.SubFactory(ExperimentFactory)
     changed_by = factory.SubFactory(UserFactory)
     old_status = factory.LazyAttribute(
-        lambda o: random.choice(Experiment.STATUS_CHOICES)[0]
+        lambda o: random.choice(NORMANDY_STATUS_CHOICES)[0]
     )
```

Comment:
Oh I see... yeah.... Actually yeah this is well in line with me splitting the factories up for normandy and nimbus.